jaeger-core/src/main/java/com/uber/jaeger/Tracer.java

```diff
@@ -370,8 +371,23 @@ Builder withMetrics(Metrics metrics) {
       return this;
     }
 
+    Builder withTag(String key, String value) {
```

Comment:
This should be public. I will change it in https://github.com/uber/jaeger-client-java/pull/142 in order to test it when corverting to thrift model.